docs/dataset-info/delivery-summary.json

```diff
@@ -1,40 +1,70 @@
+            "UofMD-landState-3-1-1"
+        ],
+        "url": "https://luh.umd.edu"
+    },
+    "ghg_concentrations": {
+        "source_ids": [
+            "CR-CMIP-1-0-0"
+        ]
+    },
+    "co2_isotopes": {
+        "source_ids": [
+            "ImperialCollege-3-0"
+        ]
+    },
+    "volcanic": {
+        "source_ids": [
+            "UOEXETER-CMIP-2-2-1"
         ]
     },
-    "biomass_burning_emms": { "source_ids": [ "DRES-CMIP-BB4CMIP7-2-0" ], "url": "https://www.globalfiredata.org" },
-    "land_use": { "source_ids": [ "UofMD-landState-3-1-1" ], "url": "https://luh.umd.edu" },
-    "ghg_concentrations": { "source_ids": [ "CR-CMIP-1-0-0" ] },
-    "co2_isotopes": { "source_ids": [ "ImperialCollege-3-0" ] },
-    "volcanic": { "source_ids": [ "UOEXETER-CMIP-2-2-1" ] },
     "ozone": {
-        "expected_publication": "piControl - end of March, historical - end of May.",
+        "expected_publication": "piControl - July 2025, historical - August 2025",
         "source_ids": null,
-        "status": "Awaiting finalization of upstream forcings: solar, volcanic, GHG/ODS concentrations, SLCF emissions, land-use changes",
+        "status": "Required upstream forcing finalization to commence simulations: solar, volcanic, GHG/ODS concentrations, SLCF emissions, and land-use changes",
         "url": null
     },
     "nitrogen-deposition": {
-        "expected_publication": "piControl - end of March, historical - end of May.",
+        "expected_publication": "piControl - July 2025, historical - August 2025",
         "source_ids": null,
-        "status": "Awaiting finalization of upstream forcings: solar, volcanic, GHG/ODS concentrations, SLCF emissions, land-use changes",
+        "status": "Required upstream forcing finalization to commence simulations: solar, volcanic, GHG/ODS concentrations, SLCF emissions, and land-use changes",
         "url": null
     },
-    "solar": { "source_ids": [ "SOLARIS-HEPPA-CMIP-4-6" ] },
-    "amip-ssts": { 
-        "source_ids": [ "PCMDI-AMIP-1-1-9" ],  
-        "status": "Final v1 dataset available. v2 dataset awaiting HadISST v2.4 release."
+    "solar": {
+        "source_ids": [
+            "SOLARIS-HEPPA-CMIP-4-6"
+        ]
+    },
+    "amip-ssts": {
+        "source_ids": [
+            "PCMDI-AMIP-1-1-9"
+        ],
+        "status": "Final v1/CMIP6Plus dataset available. v2 dataset awaiting HadISST v2.4 release."
     },
     "simple-plumes": {
         "source_ids": null,
         "url": "https://zenodo.org/records/15283189"
     },
     "population": {
-        "expected_publication": "Unknown: data provider TBD",
-        "source_ids": null,
-        "status": "Unknown: data provider TBD",
-        "url": null
+        "expected_publication": "historical - August 2025",
+        "source_ids": [
+            "PIK-CMIP-1-0-0"
+        ],
+        "status": "registered",
+        "url": "https://github.com/HORIZON-COMPASS/Exposure-and-vulnerability-modelling"
```

Comment:
```suggestion
        "source_ids": [
            "PIK-CMIP-1-0-0"
        ]
```

Now that the data is published, once #264 is merged we can simplify to this (all the other info is extracted from the data and ESGF scrape)